code/lib/core-server/src/utils/StoryIndexGenerator.test.ts

```diff
@@ -292,51 +291,6 @@ describe('StoryIndexGenerator', () => {
       });
     });
 
-    describe('mdx tagged components', () => {
-      it('adds docs entry with docs enabled', async () => {
```

Comment:
@shilman I need assistance here, I am not sure if we can just remove this test as it was targetting .stories.mdx, or if we need to test it differently with a new .mdx stub

Reply:
Let's get rid of it